kolibri/core/assets/src/vue/nav-bar/nav-bar-item.vue

```diff
@@ -21,7 +21,11 @@
   module.exports = {
     methods: {
       blur() {
-        this.$refs.link.blur();
+        if (this.$refs.link.$el) {
```

Comment:
Okay, I think I see what's happening. It encounters the second case from the `<a>` link.

I think we can actually remove the blur handler for there and update the function to be just:

```javascript
      blur() {
        this.$refs.link.$el.blur();
      },
```

because once you've clicked a link, current keyboard focus is blurred by the browser